src/applications/check-in/hooks/useFormRouting.jsx

```diff
@@ -64,9 +68,18 @@ const useFormRouting = (router = {}) => {
       const here = getCurrentPageFromRouter();
       const currentPageIndex = pages.findIndex(page => page === here);
       const nextPage = pages[currentPageIndex + 1] ?? URLS.ERROR;
-      router.push(nextPage);
+      const query = {
```

Comment:
Added always keeping the UUID in the URL so it doesn't get lost. We could also use to rebuild the current session on the page. This the start of a quality of life feature.